libs/dh/metering-point/feature-tabs/src/lib/processes-tab-content/processes-table/dh-table-util.spec.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  compareSortValues,
+  getRowToExpand,
+  wrapInTableRow,
+} from './dh-table-util';
+
+describe(wrapInTableRow.name, () => {
+  it('should wrap in table row', () => {
+    const data: string[] = ['Test data'];
+
+    const wrappedTableRow = wrapInTableRow<string>(data);
+
+    expect(wrappedTableRow.length).toBe(1);
+    expect(wrappedTableRow[0].data).toBe(data[0]);
```

Comment:
suggestion: consider testing the whole object returned by `wrapInTableRow`. I.e. `wrapInTableRow` is called with `data` and the assertion will test that `wrappedTableRow[0]` is an object with specific properties.

Reply:
Like so?

Reply:
Was thinking more like ... :)

```suggestion
    const data: string[] = ['Test data'];

    const actualValue = wrapInTableRow<string>(data);

    const expectedValue = [{
      data,
      expanded: false,
      maxHeight: 0,
    }];
  
    expect(expectedValue).toEqual(actualValue);
```